log/otfields/otfields.go

```diff
@@ -0,0 +1,39 @@
+package otfields
```

Comment:
copy of https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/lib/log/otfields/otfields.go.  Right now we're just using `Resource.`